compiler/test/runnable/xtest46.d

```diff
@@ -2583,7 +2583,7 @@ struct Foo126
 void test796()
 {
     struct S { invariant() { throw new Exception(""); } }
-    S* s;
+    S* s = new S;
     try {
         assert(s);
     } catch (Error) {
```

Comment:
This'll fail though as an Error wouldn't be raised.

Reply:
Yes the CI is failing due to that.